src/dispatch/messaging.py

```diff
@@ -364,9 +367,7 @@ class MessageType(str, Enum):
 INCIDENT_NOTIFICATION_COMMON = [INCIDENT_NAME, INCIDENT_TITLE]
 
 INCIDENT_NOTIFICATION = INCIDENT_NOTIFICATION_COMMON.copy()
-INCIDENT_NOTIFICATION.extend(
-    [INCIDENT_STATUS, INCIDENT_PRIORITY_FYI, INCIDENT_COMMANDER, INCIDENT_GET_INVOLVED_BUTTON]
```

Comment:
Remove `INCIDENT_GET_INVOLVED_BUTTON` if we're not going to use it anymore.

Reply:
I left it as I thought it might still be useful in the future. But will delete.

Reply:
Actually, this commit does remove the button?

Reply:
I meant, removing the `INCIDENT_GET_INVOLVED_BUTTON` `dict` and the `INCIDENT_GET_INVOLVED_BUTTON_DESCRIPTION`, but if you want to keep them in case they're needed in the future, fine by me.